flight/apps/data_handler.py

```diff
@@ -93,7 +93,7 @@ def __init__(
         persistent: bool = True,
         line_limit: int = 1000,
         new_config_file: bool = False,
-        home_path: str = "/sd",
```

Comment:
These all need to get changed back to /sd as we want to make sure hardware is working. Will come up with a more permanent fix for this later.

Reply:
same with al other changed instances in this file